### PR TITLE
Add label for person fields

### LINKED ISSUE #301: [PE-D2] Label of information in contact book
Screenshot 2021-11-05 at 7 13 09 PM

Maybe we should add the labelling (e.g. name: , phone: ...) in front of the details, if not hard to tell what is this information

### PR DESCRIPTION
**Pull request by e0543978**
Fixes #301 

image

### CODE REVIEW

**Review by choonyongchan — DISMISSED** [review]
Great work!

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -71,23 +71,23 @@ public PersonCard(Person person, int displayedIndex) {
...
         if (person.getEmail().value.isEmpty()) {
             email.setManaged(false);
         }
-        tutorialGroup.setText(person.getTutorialGroup().value);
+        tutorialGroup.setText("Tutorial Group: " + person.getTutorialGroup().value);
         if (person.getTutorialGroup().value.isEmpty()) {
             tutorialGroup.setManaged(false);
         }
-        remark.setText(person.getRemark().value);
+        remark.setText("Remarks: " + person.getRemark().value);
```
Comment (choonyongchan) [inline src/main/java/seedu/address/ui/PersonCard.java:90]: Just to check: For now, there isn't a case where "Remarks: "/"Phone: "/etc.. shows up right? Maybe after delete or edit?

Comment (e0543978) [reply src/main/java/seedu/address/ui/PersonCard.java:90]: I just tested with the modify functions and it looks okay! It behaves the same way as before, if the field is not included, the field label also doesn't appear.

**Review by bklimey — APPROVED** [review]
LGTM!

**Pull request merged**